app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from typing import List
+
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: List[Player]) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
Pass a generator expression instead of list comprehension to `sum` as mentioned [in the checklist](https://github.com/mate-academy/py-elves-and-dwarves/blob/master/checklist.md#code-efficiency).